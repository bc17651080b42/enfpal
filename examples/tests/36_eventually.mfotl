EVENTUALLY[0,10] B(1)