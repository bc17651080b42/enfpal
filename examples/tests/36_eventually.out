The formula
 ◊[0s,10s] B(1)
is enforceable and types to
 ◊[0s,10s] (B(1) : Cau) : Cau
[Enforcer] @1 OK.
[Enforcer] @1 nothing to do proactively.
[Enforcer] @2 OK.
[Enforcer] @2 nothing to do proactively.
[Enforcer] @3 OK.
[Enforcer] @3 nothing to do proactively.
[Enforcer] @4 nothing to do proactively.
[Enforcer] @5 nothing to do proactively.
[Enforcer] @6 nothing to do proactively.
[Enforcer] @7 nothing to do proactively.
[Enforcer] @8 nothing to do proactively.
[Enforcer] @9 nothing to do proactively.
[Enforcer] @10 nothing to do proactively.
[Enforcer] @11 proactively commands:
Cause: 
B(1)
OK.